TestCases/vandv/rans/30p30n/config.cfg

```diff
@@ -0,0 +1,94 @@
+% ------------------------------- SOLVER -------------------------------- %
+%
+SOLVER= RANS
+KIND_TURB_MODEL= SA
+REF_DIMENSIONALIZATION= FREESTREAM_VEL_EQ_MACH
+%
+% ----------------------------- FREESTREAM ------------------------------ %
+%
+MACH_NUMBER= 0.17
+AOA= 5.5
+INIT_OPTION= REYNOLDS
+FREESTREAM_OPTION= TEMPERATURE_FS
+FREESTREAM_TEMPERATURE= 295.56
+REYNOLDS_NUMBER= 1.71E6
+REYNOLDS_LENGTH= 0.4572
+%
+% -------------------------- REFERENCE VALUES --------------------------- %
+%
+REF_AREA= 0.4572
+REF_LENGTH= 0.4572
+REF_ORIGIN_MOMENT_X= 0.0
+REF_ORIGIN_MOMENT_Y= 0.0
+REF_ORIGIN_MOMENT_Z= 0.0
+%
+% ------------------------- BOUNDARY CONDITIONS ------------------------- %
+%
+MARKER_HEATFLUX= ( wall, 0.0 )
+MARKER_FAR= ( farfield )
+MARKER_PLOTTING= ( wall )
+MARKER_MONITORING= ( wall )
+%
+% ---------------------------- FLUID MODELS ----------------------------- %
+%
+FLUID_MODEL= STANDARD_AIR
+GAMMA_VALUE= 1.4
+GAS_CONSTANT= 287.058
+%
+VISCOSITY_MODEL= SUTHERLAND
+MU_REF= 1.716E-5
+MU_T_REF= 273.15
+SUTHERLAND_CONSTANT= 110.4
+%
+CONDUCTIVITY_MODEL= CONSTANT_PRANDTL
+PRANDTL_LAM= 0.72
+PRANDTL_TURB= 0.90
+%
+% ----------------------- SPATIAL DISCRETIZATION ------------------------ %
+%
+NUM_METHOD_GRAD= GREEN_GAUSS
+CONV_NUM_METHOD_FLOW= ROE
+ENTROPY_FIX_COEFF= 1e-5
+MUSCL_FLOW= YES
+SLOPE_LIMITER_FLOW= VAN_ALBADA_EDGE
+%
+CONV_NUM_METHOD_TURB= SCALAR_UPWIND
+MUSCL_TURB= NO
+%
+% ---------- PSEUDOTIME INTEGRATION / CONVERGENCE ACCELERATION ---------- %
+%
+USE_VECTORIZATION= YES
+TIME_DISCRE_FLOW= EULER_IMPLICIT
+TIME_DISCRE_TURB= EULER_IMPLICIT
+%
+CFL_NUMBER= 40
+CFL_REDUCTION_TURB= 1.0
+CFL_ADAPT= NO
+%
+LINEAR_SOLVER= FGMRES
+LINEAR_SOLVER_PREC= ILU
+LINEAR_SOLVER_ERROR= 0.2
+LINEAR_SOLVER_ITER= 4
+%
+MGLEVEL= 0
+NEWTON_KRYLOV= YES
+NEWTON_KRYLOV_IPARAM= ( 0, 0, 1 ) % n0, np, ft
+NEWTON_KRYLOV_DPARAM= ( 0.0, 1e-20, -3, 1e-5 ) % r0, tp, rf, e
+%
+% ------------------------ CONVERGENCE CRITERIA ------------------------- %
+%
+ITER= 10000
+CONV_RESIDUAL_MINVAL= -11.5
+%
+% --------------------------- INPUT / OUTPUT ---------------------------- %
+%
+MESH_FILENAME= 2D_L1_coarse_r1.su2
```

Comment:
Concerning https://github.com/su2code/VandV/commit/f5a89277dbf61616c9c0d11524939e522a61934f ... where do you unzip the mesh files?

Reply:
I don't. I cannot change the machines running the tests, therefore I cannot make them checkout the vandv repo, thus I must add what I need to run the tests in Testcases.